docs/guides/getting-started-orchestrate-api-endpoints.md

```diff
@@ -19,15 +19,13 @@ To use a **REST Connector** in your process, follow the steps below:
 
 1. Create a BPMN diagram. To do this, navigate to Web Modeler via the **Modeler** tab, and click **New project**.
 2. Name your project and select **New > BPMN Diagram > + Create blank**.
-3. Give your model a descriptive name and id. On the right side of the page, expand the **General** section of the properties panel to find the name and id fields. For this guide, we'll use **API Endpoint Tutorial** for the name and **api-endpoint-tutorial** for the id.
+3. Give your model a descriptive name and id. On the right side of the page, expand the **General** section of the properties panel to find the name and id fields. For this guide, we'll use `API Endpoint Tutorial` for the name and `api-endpoint-tutorial` for the id.
 4. Use Web Modeler to design a BPMN flow with a Connector task. Create a Connector task by dragging the rectangular task element from the palette, or click the existing start event and the displayed task element to the right of the start event.
-5. Change the task type by clicking the wrench icon and select **REST Connector (No Auth)** in the **Connectors** section.
+5. Change the task type by clicking the wrench icon and select **REST Connector (No Auth)** in the **Connectors** section. Alternatively, you can directly choose a **REST Connector** by using the **append menu**.
```

Comment:
Is it lower case and named in the product?

Reply:
I assume you are taking about the "REST Connector". It is referred like this in the product itself. 
![image](https://user-images.githubusercontent.com/30292227/197133675-97a03e89-4062-4870-bd5f-09f134faf0c9.png)
If you are talking about the "append menu" - It does not feature a label in the product but has been there before I changed anything.

Reply:
Do you mean the context pad in this case according to the [glossary](https://confluence.camunda.com/display/HAN/Components+Glossary)? Regardless, there's no need to bold this wording if it isn't directly written out in the UI.

Reply:
Yes exactly! It is the context pad - thanks @christinaausley!